pkg/getty/Dockerfile

```diff
@@ -9,6 +9,19 @@ RUN apk add --no-cache --initdb -p /out \
     util-linux \
     && true
 RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
+#
+# when run standalone as a regular or services container, the below is unnecessary
+#  when run as a linuxkit init, all files are copied over
+#  if a different image is later in the sequence, that one may use busybox,
+#    which symlinks `setsid -> /bin/busybox`. That would override `setsid` and cause this to fail,
+#    since `setsid` from busybox lacks the `-w` option, which `linux-utils` has.
+#  we cp setsid to a normally unused location so we are guaranteed it sticks around,
+#    and then reference it by the unique `setsid.getty` name in `rungetty.sh`.
+#  if linuxkit.yml init were composed entirely of images that do not override `setsid`, we
+#    could remove this line and use just `setsid` in `rungetty.sh`
+RUN cp /out/usr/bin/setsid /out/usr/bin/setsid.getty
```

Comment:
I think you can simplify this explanation a bit e.g. :
```
# We require a version of `setsid(1)` which supports the `-w`
# option, which is not available in all implementations (e.g. the
# `busybox` implementation does not support it). When this is run
# as part of a LinuxKit `init` image (rather than as a standalone
# container) we cannot guarantee which version of `setsid` will
# be present once the layers are combined, so we take a copy of
# our own, known good, version for use later.
```